libs/components/tiles/src/lib/modules/tiles/tile/tile-title.component.html

```diff
@@ -5,5 +5,8 @@
     'sky-section-heading': 'default'
   }"
 >
-  <ng-content></ng-content>
+  <span skyTrim><ng-content></ng-content></span
+  ><span class="sky-control-help-container"
```

Comment:
We don't need to add margin or styles or anything?

Reply:
The `sky-control-help-container` class handles that and minimizes component-specific implementation code.

Reply:
Cool, didn't realize that was a thing. Nice!